From d924c3bf4d83e9ef3ce66a6ac1e80ef1cb7cc4ca Mon Sep 17 00:00:00 2001
From: Ruslan Morozov <ruslan.y.morozov@gmail.com>
Date: Sat, 3 Jun 2023 12:21:15 +0300
Subject: [PATCH] [PATCH] fix skip install rules

---
 include/BoostRoot.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/cmake/include/BoostRoot.cmake b/tools/cmake/include/BoostRoot.cmake
index e93f907..f0380b3 100644
--- a/tools/cmake/include/BoostRoot.cmake
+++ b/tools/cmake/include/BoostRoot.cmake
@@ -108,7 +108,7 @@ else()
   endif()
 
   set(BUILD_TESTING OFF)
-  set(BOOST_SKIP_INSTALL_RULES ON)
+  set(BOOST_SKIP_INSTALL_RULES OFF)
 
 endif()
 
-- 
2.39.2 (Apple Git-143)

